meson.build

```diff
@@ -264,6 +264,15 @@ elif host_machine.system() == 'dripos'
 	rtld_include_dirs += include_directories('sysdeps/dripos/include')
 	libc_include_dirs += include_directories('sysdeps/dripos/include')
 	subdir('sysdeps/dripos')
+elif host_machine.system() == 'astral'
+        disable_crypt_option = true
+        disable_iconv_option = true
+        disable_intl_option = true
```

Comment:
I would leave these settings up to the configurer of mlibc rather than hardcoding them here. It also would align with the other ports.